packages/neutron-sdk/src/interchain_queries/types.rs

```diff
@@ -187,6 +210,259 @@ impl KVReconstruct for Balances {
     }
 }
 
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+/// A structure that can be reconstructed from **StorageValues**'s for the **Bank Total Interchain Query**.
+/// Contains total supply for specific denom that are held on remote chain.
+pub struct TotalSupply {
```

Comment:
What about tests for all these new structures (`TotalSupply`, `GovernmentProposal`, `StakingValidator`, `FeePool`)?